<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdWindow</class>
 <widget class="QMainWindow" name="QxrdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QXRD Readout Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Image</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QxrdImagePlot" name="m_Plot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
          </widget>
          <widget class="QToolBox" name="toolBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>300</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>378</width>
              <height>656</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Acquire</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QWidget" name="widget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Exposure Time:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QComboBox" name="m_ExposureTime">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Summed Frames:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QSpinBox" name="m_SummedFrames">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Frames in sequence:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QSpinBox" name="m_SequenceFrames">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0" colspan="2">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Output Directory:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0" colspan="2">
                 <widget class="QLineEdit" name="m_OutputDirectory">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0" colspan="2">
                 <widget class="QPushButton" name="m_SelectDirectoryButton">
                  <property name="text">
                   <string>Select Output Directory...</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0" colspan="2">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Save File Pattern:</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0" colspan="2">
                 <widget class="QLineEdit" name="m_SaveFilePattern">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0" colspan="2">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>File Index Number:</string>
                  </property>
                 </widget>
                </item>
                <item row="12" column="0" colspan="2">
                 <widget class="QSpinBox" name="m_FileIndexNumber">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
                <item row="13" column="0">
                 <widget class="QPushButton" name="m_CancelButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
                <item row="13" column="1">
                 <widget class="QPushButton" name="m_AcquireButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Acquire</string>
                  </property>
                 </widget>
                </item>
                <item row="15" column="0" colspan="2">
                 <widget class="QSpinBox" name="m_DarkSummedFrames">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>99999</number>
                  </property>
                 </widget>
                </item>
                <item row="14" column="0" colspan="2">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Summed Frames for Dark Measurement:</string>
                  </property>
                 </widget>
                </item>
                <item row="16" column="1">
                 <widget class="QPushButton" name="m_DarkAcquireButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Dark</string>
                  </property>
                 </widget>
                </item>
                <item row="16" column="0">
                 <widget class="QPushButton" name="m_DarkCancelButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>237</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>378</width>
              <height>656</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Correction</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Dark Subtraction</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0" colspan="2">
                 <widget class="QCheckBox" name="m_PerformDark">
                  <property name="text">
                   <string>Perform Dark Subtraction during Acquisition</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QCheckBox" name="m_SaveRaw">
                  <property name="text">
                   <string>Save 'raw' images (as well as corrected)</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Dark Image file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLineEdit" name="m_DarkImageName">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>188</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="4" column="1">
                 <widget class="QPushButton" name="m_LoadDarkButton">
                  <property name="text">
                   <string>Load Dark</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Bad Pixel Correction</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0" colspan="2">
                 <widget class="QCheckBox" name="m_PerformBadPixels">
                  <property name="text">
                   <string>Perform Bad Pixel Correction during Acquisition</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Bad Pixel file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLineEdit" name="m_BadPixelsFileName">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>188</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="m_LoadBadPixelsButton">
                  <property name="text">
                   <string>Load Bad Pixels</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Gain Correction</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0" colspan="2">
                 <widget class="QCheckBox" name="m_PerformGainCorrection">
                  <property name="text">
                   <string>Perform Gain Correction during Acquisition</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Gain correction file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QLineEdit" name="m_GainCorrectionFileName">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>188</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="m_LoadGainCorrection">
                  <property name="text">
                   <string>Load Gain Map</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>212</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>378</width>
              <height>656</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Display</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="m_Messages"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_ActionLoadData"/>
    <addaction name="m_ActionSaveData"/>
    <addaction name="m_ActionEvaluateScript"/>
    <addaction name="m_ActionPreferences"/>
    <addaction name="m_ActionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAcquire">
    <property name="title">
     <string>Acquire</string>
    </property>
    <addaction name="m_ActionAcquire"/>
    <addaction name="m_ActionAcquireDark"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="m_ActionAutoScale"/>
    <addaction name="separator"/>
    <addaction name="m_Action005Range"/>
    <addaction name="m_Action010Range"/>
    <addaction name="m_Action100Range"/>
    <addaction name="separator"/>
    <addaction name="m_ActionGrayscale"/>
    <addaction name="m_ActionInverseGrayscale"/>
    <addaction name="m_ActionEarthTones"/>
    <addaction name="m_ActionSpectrum"/>
    <addaction name="m_ActionFire"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAcquire"/>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_ActionAcquire"/>
   <addaction name="m_ActionCancel"/>
   <addaction name="m_ActionAcquireDark"/>
   <addaction name="m_ActionCancelDark"/>
   <addaction name="m_ActionEvaluateScript"/>
   <addaction name="separator"/>
   <addaction name="m_ActionLoadData"/>
   <addaction name="m_ActionSaveData"/>
   <addaction name="separator"/>
   <addaction name="m_ActionAutoScale"/>
   <addaction name="m_Action005Range"/>
   <addaction name="m_Action010Range"/>
   <addaction name="m_Action100Range"/>
   <addaction name="separator"/>
   <addaction name="m_ActionGrayscale"/>
   <addaction name="m_ActionInverseGrayscale"/>
   <addaction name="m_ActionEarthTones"/>
   <addaction name="m_ActionSpectrum"/>
   <addaction name="m_ActionFire"/>
   <addaction name="separator"/>
   <addaction name="m_ActionQuit"/>
  </widget>
  <action name="m_ActionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="m_ActionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="m_ActionAcquire">
   <property name="text">
    <string>Acquire</string>
   </property>
  </action>
  <action name="m_ActionAutoScale">
   <property name="text">
    <string>AutoScale</string>
   </property>
  </action>
  <action name="m_ActionLoadData">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="m_ActionSaveData">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="m_ActionEvaluateScript">
   <property name="text">
    <string>Evaluate Script</string>
   </property>
  </action>
  <action name="m_ActionCancel">
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="m_ActionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="m_ActionInverseGrayscale">
   <property name="text">
    <string>Inverse Grayscale</string>
   </property>
   <property name="toolTip">
    <string>Inverse Grayscale</string>
   </property>
  </action>
  <action name="m_ActionEarthTones">
   <property name="text">
    <string>Earth Tones</string>
   </property>
  </action>
  <action name="m_ActionSpectrum">
   <property name="text">
    <string>Spectrum</string>
   </property>
  </action>
  <action name="m_ActionFire">
   <property name="text">
    <string>Fire</string>
   </property>
  </action>
  <action name="m_Action010Range">
   <property name="text">
    <string>10 % Range</string>
   </property>
  </action>
  <action name="m_Action005Range">
   <property name="text">
    <string>5 % Range</string>
   </property>
  </action>
  <action name="m_Action100Range">
   <property name="text">
    <string>100% Range</string>
   </property>
  </action>
  <action name="m_ActionAcquireDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="m_ActionCancelDark">
   <property name="text">
    <string>Cancel Dark</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxrdImagePlot</class>
   <extends>QwtPlot</extends>
   <header>qxrdimageplot.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>m_ExposureTime</tabstop>
  <tabstop>m_SummedFrames</tabstop>
  <tabstop>m_SequenceFrames</tabstop>
  <tabstop>m_OutputDirectory</tabstop>
  <tabstop>m_SelectDirectoryButton</tabstop>
  <tabstop>m_SaveFilePattern</tabstop>
  <tabstop>m_FileIndexNumber</tabstop>
  <tabstop>m_DarkSummedFrames</tabstop>
  <tabstop>m_AcquireButton</tabstop>
  <tabstop>m_CancelButton</tabstop>
  <tabstop>m_DarkAcquireButton</tabstop>
  <tabstop>m_DarkCancelButton</tabstop>
  <tabstop>m_Messages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
